projects/swimlane/ngx-ui/src/lib/components/tooltip/tooltip.directive.ts

```diff
@@ -20,7 +20,7 @@ import { TooltipContentComponent } from './tooltip.component';
 import { TooltipService } from './tooltip.service';
```

Comment:
TooltipContentComponent is an unused import here

Reply:
it's actually needed by the `InjectionRegisteryService` it's inheriting from, but i added a type to it instead of any